docs/core/entity/vacuum.md

```diff
@@ -11,10 +11,6 @@ Derive entity platforms from [`homeassistant.components.vacuum.StateVacuumEntity
 Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
 :::
 
-:::note
-`VacuumEntity` is deprecated and will be removed in future releases. Please use or migrate to the `StateVacuumEntity`
```

Comment:
I noticed that we still have some pylint plugin code in core that references `VacuumEntity`.

Reply:
I only checked for integrations but I'll make a pass and see what possible references we have lying around.